src/input_file.rst

```diff
@@ -730,6 +730,16 @@ In this section, input file names are listed. In addition, if you want to restar
     Structure   0   ISB_T_270_k_merged.psf
     Structure   1   ISB_T_270_k_merged.psf
 
+``MultiSimFolderName (MPI Compilation Required)``
```

Comment:
`(MPI Compilation Required)` is not part of keyword. You need to take it out.

Reply:
![image](https://user-images.githubusercontent.com/39970712/72402381-12f75600-371d-11ea-897c-90282fa8b712.png)